crates/evm/evm/src/executors/corpus.rs

```diff
@@ -628,6 +626,25 @@ impl CorpusManager {
 
         tx.call_details.calldata =
             function.abi_encode_input(&prev_inputs).map_err(|e| eyre!(e.to_string()))?.into();
+
+        // Mutate sender with 15% probability
+        if test_runner.rng().random_ratio(15, 100) {
+            tx.sender = Address::random();
+        }
+
+        // Mutate value with 15% probability
+        if test_runner.rng().random_ratio(15, 100) {
+            // Generate random value, biased towards smaller values for practical fuzzing
+            let value = match test_runner.rng().random_range(0..=10) {
+                0..=5 => U256::from(test_runner.rng().random_range(0..=1000)), // Small values
+                6..=8 => U256::from(test_runner.rng().random_range(0..=1_000_000_000_000_000u64)), // Medium values (up to 0.001 ETH)
+                9 => U256::from(test_runner.rng().random_range(0..=1_000_000_000_000_000_000u64)), // Large values (up to 1 ETH)
+                10 => U256::MAX, // Edge case
+                _ => unreachable!(),
+            };
+            tx.call_details.value = Some(value);
+        }
```

Comment:
This should use the existing U256 [mutator](https://github.com/foundry-rs/foundry/blob/fdf5732d08ce1c67aa0aaf047c3fb86614caf5ae/crates/evm/fuzz/src/strategies/param.rs#L284) and be added as a mutation [here](https://github.com/foundry-rs/foundry/blob/fdf5732d08ce1c67aa0aaf047c3fb86614caf5ae/crates/evm/evm/src/executors/corpus.rs#L376)



I believe you will also need to add the balance to the sender in the DB